blockchain/reactor.go

```diff
@@ -163,6 +162,12 @@ func (bcR *BlockchainReactor) Receive(chID byte, src *p2p.Peer, msgBytes []byte)
 	}
 }
 
+// maxMsgSize returns the maximum allowable size of a
+// message on the blockchain reactor.
+func (bcR *BlockchainReactor) maxMsgSize() int {
+	return bcR.state.Params().MaxBlockSizeBytes + 2
```

Comment:
What's the magic number 2 for?

Reply:
The extra encoding that happens to turn the block into a BlockchainMessage. I think it might actually be `+1` though. We should add a test for this stuff.

Reply:
Roger that, and also perhaps a variable ;)